### PR TITLE
feat(extension): added funding layout

### PR DESCRIPTION
**Pull request by vladdezhniuk**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by NikolaiYurchenko — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <Wrapper justifyContent='space-between'>
+      <PrefixSuffix onClick={onPrefixClick}>{prefix}</PrefixSuffix>
+      {children}
+      <PrefixSuffix onClick={onSuffixClick}>{suffix}</PrefixSuffix>
+    </Wrapper>
+  )
+}
+
+export default DefaultHeader
```
Comment (VitaliiShpital) [inline packages/blockchain-wallet-v4-frontend/src/scenes/plugin/DefaultHeader.tsx]: unrelated change?
